benches/benches/standard.rs

```diff
@@ -27,7 +27,7 @@ where
 {
     g.throughput(criterion::Throughput::Bytes(size_of::<T>() as u64));
     g.bench_function(name, |b| {
-        let mut rng = Pcg64Mcg::from_os_rng();
+        let mut rng = Pcg64Mcg::from_rng(&mut rand::rng());
```

Comment:
Use of `from_os_rng` should not influence the benchmark results since only time for the closure passed to `iter` is measured.

Reply:
Agreed. It may improve set-up time slightly; at any rate I see no harm in using `rand::rng()`.